src/background.js

```diff
@@ -65,20 +76,30 @@ function handleWake() {
   const now = Date.now();
   log('woke at', now);
   return browser.storage.local.get().then(items => {
-    const due = Object.values(items).filter(item => item.time <= now);
+    const due = Object.entries(items).filter(entry => entry[1].time <= now);
     log('tabs due to wake', due.length);
-    return Promise.all(due.map(item =>
-      browser.tabs.create({
-        active: false,
-        url: item.url,
-        windowId: item.windowId
-      }).then(tab => browser.notifications.create(`${item.windowId}:${tab.id}`, {
-        'type': 'basic',
-        'iconUrl': browser.extension.getURL('link.png'),
-        'title': item.title,
-        'message': item.url
-      })).then(() => browser.storage.local.remove(idForItem(item)))
-    ));
+    return browser.windows.getAll({
```

Comment:
Thinking through this bit so I understand it: Looks like it reuses windows by ID when they exist, but open new ones otherwise? Not sure what it did before, so this seems like a nice bugfix.